apps/Test App/Test App/Mux Style/Mux Colors.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  MuxColors.swift
+//  Test App
+//
+//  Created by Emily Dixon on 5/10/23.
+//
+
+import Foundation
+import SwiftUI
+
+// Mux Grays
+let Gray100 = Color(red: 10 / CONVERSION_CSS, green: 10 / CONVERSION_CSS, blue: 11 / CONVERSION_CSS)
```

Comment:
You can define these as named colors in the app asset catalog, they're then accessible using a [similar API](https://developer.apple.com/documentation/uikit/uicolor/2877380-init) to asset catalog derived `UIImage`. This might make it easier to update them later.

Reply:
Also here's a tool I use to convert between HEX values and RGB colors: https://www.uicolor.io

Reply:
Are Asset Catalogs easy to port between apps? This would be a must, as I intend to copy all this stuff into our next example app ;)

Reply:
Yeah you can copy and paste anything in the asset catalog between Xcode projects